pkgs/os-specific/linux/fxload/default.nix

```diff
@@ -1,22 +1,15 @@
-{lib, stdenv, fetchurl}:
+{lib, stdenv, fetchgit}:
 
 stdenv.mkDerivation rec {
   pname = "fxload";
-  version = "2002.04.11";
+  version = "2013-01-03";
```

Comment:
Please update the commit message as well.

```suggestion
  version = "unstable-2013-01-03";
```

Reply:
Update it to what?

Reply:
Never mind, forgot this wasn't a package addition.

Reply:
a'ight!